lib/presentation/pages/login/controller/login_controller.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/cupertino.dart';
+import 'package:get/get.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:task/core/preferences.dart';
+import 'package:task/core/routes.dart';
+import 'package:task/data/models/onBoard/user/user_model.dart';
+
+class LoginController extends GetxController {
```

Comment:
First declare the variable the  use them 
why oninit is coming first and then all variables are declaring

Reply:
OK